### DISCUSSION TITLE
Unavailable usernames even though they're available

### DISCUSSION THREAD

**Discussion by donno2048** [community/community#33768] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/61805754/191624840-af0156b0-38b0-4989-9cb6-bda5ecf074e8.png)

![image](https://user-images.githubusercontent.com/61805754/191624857-83b0b173-e946-410c-975e-134a6d610d61.png)

![image](https://user-images.githubusercontent.com/61805754/191624866-d5f687f3-84fe-45cd-bcd6-c65688b85237.png)

**Comment by Sagar-Sharma-7** [comment]
I guess, the username should be of at least 6 characters.

**Reply by donno2048** [reply]
![image](https://user-images.githubusercontent.com/61805754/191696284-bbb8dab4-0dd3-4a85-a1e2-fe5f7f65fad7.png)

**Reply by jfullstackdev** [reply]
if you look at the user's timeline, you'll see the account was last 2012, maybe at that time, it was allowed

![image](https://user-images.githubusercontent.com/47092464/191697696-964c61e9-b9c9-4160-87bb-f444d86049c2.png)

**Reply by donno2048** [reply]
![image](https://user-images.githubusercontent.com/61805754/191716786-a0e60615-d210-4118-b6ec-fc706c99fbd1.png)

**Comment by jfullstackdev** [comment]
yup, I think too that it should be at least 6 characters, as mentioned here

https://github.com/phpipam/phpipam/issues/127

**Comment by billythekid** [comment]
Hi @donno2048 👋 

Not all accounts are publicly visible on GitHub and not all endpoints are available for registration as accounts.

The username change tool will let you know if a username is available.

In the case of `c` in your _tests_ there, that's actually used to direct people to our support portal (I think it used to be a shortcut for `/contact` - which also directs there)

https://github.com/c

Cheers,
Billy 🤠

**Reply by donno2048** [reply]
Thanks!
